### PR TITLE
Mvc #1

### PR DESCRIPTION
**Pull request by yannikmaassen**
Hi!
I rearranged the routing, added seeders and migrations and prepared the views to list and show the database datasets. Is it actually better to create new tables in the database for the genres and headliner? I think my current attempt could definitely need improvement.

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -23,10 +23,10 @@
 Route::get('/profile/edit', 'ProfileController@edit')->name('profile.edit');
 Route::get('/profile/other', 'ProfileController@other')->name('profile.other');
 
-Route::get('/planner/show', 'PlannerController@show')->name('planner.show');
-Route::get('/planner/create', 'PlannerController@create')->name('planner.create');
 Route::get('/planner/overview', 'PlannerController@index')->name('planner.overview');
```
Comment (katklat) [inline routes/web.php:26]: You could make a resource controller and add your custom functions to it

File: app/Http/Controllers/PlannerController.php [hunk 2]
```diff
@@ -11,9 +12,12 @@ class PlannerController extends Controller
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(Planner $planner)
     {
-        //
+        $planner = Planner::all();
```
Comment (tillschander) [inline app/Http/Controllers/PlannerController.php:17]: Later on you should change this so it only shows the users planners and not everyone's planners.

File: app/Planner.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Planner extends Model
+{
+    protected $fillable = ['name', 'start_date', 'end_date', 'genre_1', 'genre_2', 'genre_3', 'genre_4', 'genre_5', 'headliner_1', 'headliner_2', 'headliner_3', 'headliner_4', 'headliner_5', 'description', 'todo_list', 'playlist', 'planner_image'];
```
Comment (tillschander) [inline app/Planner.php:9]: Yes, i think it would be better to create separate tables for headliners and genres.

File: routes/api.php [hunk 4]
```diff
@@ -17,3 +17,7 @@
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::get('/test', function (Request $request) {
+    return 'Hello World!';
+});
```
Comment (tillschander) [inline routes/api.php:23]: Haha best route :D

**Pull request merged**